hits_ingest.py

```diff
@@ -141,24 +153,45 @@
         print('~~~ !!! ~~~')
     else:
         print('Success!')
-        print('Calibrations corresponding to {0} are now ingested in {1}' \
-                                                    .format(repo, calibrepo))
+        print('Calibrations corresponding to {0} are now ingested in {1}'.format(repo, calibrepo))
 
-# the final step is to run processCcd to do ISR by combining the ingested
+# the next step is to run processCcd to do ISR by combining the ingested
 # images and calibration products
-if doProcessCcd:
+# astrometry and photometric calibration is also done by processCcd
+elif args.task == 'processCcd':
+    if not os.path.isdir(processedrepo):
+        os.mkdir(processedrepo)
     print('Running ProcessCcd...')
     OBS_DECAM_DIR = getPackageDir('obs_decam')  # os.getenv('OBS_DECAM_DIR')
-    # TODO: once DM-5466 is complete, the chdir should be removed and the
-    #       relative '../' path nonsense in args should be removed too
-    os.chdir(calibrepo)
-    args = ['../' + repo, '--id', 'visit=' + visit, 'ccdnum=' + ccdnum, 
-            '--output', '../' + outputrepo, 
+    config = ProcessCcdConfig()
+    # Astrometry retarget party
+    for refObjLoader in (config.calibrate.astromRefObjLoader,
+                         config.calibrate.photoRefObjLoader,
+                         config.charImage.refObjLoader,):
+        refObjLoader.retarget(LoadIndexedReferenceObjectsTask)
+        refObjLoader.ref_dataset_name = 'pan-starrs'  # options are gaia, pan-starrs, sdss
+        refObjLoader.filterMap = {"g": "g",  # 'phot_g_mean_mag' is gaia-specific
+                                  "r": "r",  # all of 'g,r,i,z,y' are options for pan-starrs
+                                  "VR": "g"}
+    args = [repo, '--id', 'visit=' + visit, 'ccdnum=' + ccdnum,
+            '--output', processedrepo,
+            '--calib', calibrepo,
             '-C', OBS_DECAM_DIR + '/config/processCcdCpIsr.py',
-            '--config', 'calibrate.doAstrometry=False',
-            'calibrate.doPhotoCal=False',
-            '--no-versions']
-    # This is SO MUCH EASIER than parsing the args in the above two cases!!
-    ProcessCcdTask.parseAndRun(args=args, doReturnResults=True)
-
-    
+            '--config', 'calibrate.doAstrometry=True',
```

Comment:
I believe they are True the default, but I'd like to argue to keep the explicit assignments because it differs from the example in the obs_decam README which has them both explicitly set to False. For difference imaging to work, they need to both be True.